enhancements/network/cloud-egress-ip.md

```diff
@@ -0,0 +1,445 @@
+---
+title: TBD - for now: Cloud API component for egress IP
+authors:
+  - "@alexanderConstantinescu"
+reviewers:
+  - "@danwinship"
+  - "@abhat"
+  - "@squeed"
+  - "@tssurya"
+  - "@trozet"
+approvers:
+  - TBD
+creation-date: 2020-12-16
+last-updated: -
+status: implementable
+---
+
+# TBD - for now: Cloud API component for egress IP
+
+This covers adding cloud provider support for the OpenShift networking feature:
+egress IP. It is not clear today what the extent of the component presented here
```

Comment:
We should also call out that whatever be the design we propose here, should likely have elements of it that can be reused for configuring security groups and other such cloud configuration requirements - although that shouldn't be a driving force in the design choice.

Reply:
As mentioned here: https://github.com/openshift/enhancements/pull/571#discussion_r547493387 I think we should just design for egress IP for now. We don't have any requirements of what these other cloud configuration modifications would be, which makes it difficult to take into account now.